lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -38,7 +38,7 @@
                         <%- gettext('Students in this cohort are:') %>
```

Comment:
@catong @explorerleslie @muhammad-ammar It is slightly better without the ':' but it really shouldn't be a partial sentence. I think it should be something like "Cohort Assignment Scheme".